ert_shared/ensemble_evaluator/config.py

```diff
@@ -7,7 +7,6 @@
 CLIENT_URI = "client"
 DISPATCH_URI = "dispatch"
 DEFAULT_PORT = str(8765 + random.randint(0, 30))
-DEFAULT_HOST = "0.0.0.0"
```

Comment:
?

Reply:
As you might spot, this was not a successful operation :D

The merge is not done correctly, probably doing the inverse of what I want.

Reply:
Fixed